lib/ansible/plugins/filter/to_datetime.yml

```
@@ -0,0 +1,36 @@
+DOCUMENTATION:
+  name: to_datetime
+  version_added: "2.4"
+  short_description: Get datetime from string
+  description:
+    - Using the input string attempt to create a matching Python datetime object
+  notes:
+    - For a full list of format codes for working with python date format strings, see https://docs.python.org/3/library/datetime.html#strftime-and-strptime-behavior.
+  positional: _input, second, utc
```

The `second` and `utc` options need to be documented below, otherwise the docs build won't be happy.

---

Actually these two are not arguments for this filter, so
```suggestion
  positional: _input
```